store/storetest/mocks/ComplianceStore.go

```diff
@@ -127,19 +127,18 @@ func (_m *ComplianceStore) Save(compliance *model.Compliance) (*model.Compliance
 		}
 	}
 
-	var r1 *model.AppError
-	if rf, ok := ret.Get(1).(func(*model.Compliance) *model.AppError); ok {
+	var r1 error
+	if rf, ok := ret.Get(1).(func(*model.Compliance) error); ok {
 		r1 = rf(compliance)
 	} else {
-		if ret.Get(1) != nil {
-			r1 = ret.Get(1).(*model.AppError)
-		}
+		r1 = ret.Error(1)
 	}
 
 	return r0, r1
 }
 
 // Update provides a mock function with given fields: compliance
+// Edit
 func (_m *ComplianceStore) Update(compliance *model.Compliance) (*model.Compliance, *model.AppError) {
```

Comment:
this should be updated to return plain error i think?

but when I run make store-mocks I get

```
mattermost-server$ make store-mocks 
go get -modfile=go.tools.mod github.com/vektra/mockery/...
/Users/go/src/github.com/farah/mattermost-server/bin/mockery -dir store -all -output store/storetest/mocks -note 'Regenerate this file using `make store-mocks`.'
Error parsing file:  /Users/go/src/github.com/farah/mattermost-server/store/storetest/store.go:60:77: cannot use &s.ComplianceStore (value of type *mocks.ComplianceStore) as store.ComplianceStore value in return statement: wrong type for method Update
Error parsing file:  /Users/go/src/github.com/farah/mattermost-server/store/storetest/store.go:60:77: cannot use &s.ComplianceStore (value of type *mocks.ComplianceStore) as store.ComplianceStore value in return statement: wrong type for method Update
```